### ISSUE TITLE
PDF DotAsset should have hasTitleImage: true

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#18483]
When a contentlet have an image field (or binary) it bring two fileds to help us to show the thumbnail:

```
{
    ...
    hasTitleImage: 'true',
    titleImage: 'asset',
};
```

In the case of the DotAseet PDF thet data is not coming.

## Steps to Test:
1. Upload a PDF using the dotAsset Content Type.
2. Select Card View on the Search Portlet.
3. The PDF should show an image not the PDF icon.

## Acceptance Criteria

1. Add `hasTitleImage` and `titleImage` to PDFDotAsset.
2. Work in all the supported browsers (don't forget IE11 and iPad)
3. Multilanguage
4. Unit test

**Comment by hmoreras** [comment]
PR: https://github.com/dotCMS/dotcms-webcomponents/pull/32

**Comment by bryanboza** [comment]
Fixed, tested on release-5.3.2 / Postgres // FF
